test/resources/org/apache/jmeter/protocol/http/util/shift_jis.properties

```diff
@@ -0,0 +1,2 @@
+value=����l���s
```

Comment:
Please avoid adding "non-utf-8" files to the repository.
It complicates support.

Reply:
This was added in order to test the functionality of encoding via "shiftjis" which is added via this PR.

Reply:
The are lots of way to test encoding. You don't have to create files for that.
You could just use `new byte[]{1,2,3,4}`